app/Http/Controllers/UserController.php

```diff
@@ -103,7 +103,7 @@ public function admin_store(Request $request, $userid){
         return redirect()->action('UserController@admin', [$userid]);
     }
 
-    public function activity_index(){
+    public function activity_index() {
```

Comment:
__[code_sniffer]__
* Method name "UserController::activity_index" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->